src/script.c

```diff
@@ -387,6 +387,16 @@ static int scriptVerifyWriteCommandAllow(scriptRunCtx *run_ctx, char **err) {
     return C_OK;
 }
 
+static int scriptVerifyMayReplicate(scriptRunCtx *run_ctx, char **err) {
+    if (run_ctx->c->cmd->flags & CMD_MAY_REPLICATE &&
+        server.client_pause_type == CLIENT_PAUSE_WRITE) {
+        *err = sdsnew("May-replicate commands are not allowed when client pause write.");
```

Comment:
```suggestion
        *err = sdsnew("-ERR May-replicate commands are not allowed when client pause write.");
```

Reply:
we don't need do that, `luaPushError()` and `luaCallFunction()` already help to add the prefix

Reply:
i was looking at the surroundings functions (`scriptVerifyOOM`) and saw they include error codes, but now i see not all of them.
and also you're right, `luaPushError` and `afterErrorReply` handle that.
and also your test verifies it (could also maybe verify the error stats, but maybe that's excessive)